fleece/parameters.py

```diff
@@ -0,0 +1,130 @@
+import six
+import boto3
+
+
+def _flatten_config(config):
+    """Creates a flat list of parameters from a configuration dictionary.
+    Each element in the list is a tuple with (name, value, encrypted).
+
+    Supported values are strings, nested dictionaries and nested lists. List
```

Comment:
This will work for nested dicts only up to a point. SSM Parameter hierarchy has a maximum of 5 levels: http://docs.aws.amazon.com/systems-manager/latest/userguide/sysman-paramstore-working.html.